scripts/generate-readmes.ts

```diff
@@ -0,0 +1,184 @@
+/* eslint-disable @nangohq/custom-integrations-linting/no-try-catch-unless-explicitly-allowed */
+/* eslint-disable no-console */
+import { promises as fs } from 'fs';
+import { load as loadYaml } from 'js-yaml';
+
+const divider = '<!-- END  GENERATED CONTENT -->';
```

Comment:
there is a `<!-- END  GENERATED CONTENT -->` tag but not a beginning one. Are we assuming it always starts at the top of the file?

Reply:
Yeah, I can toss one up top.

Reply:
Fixed in 536a35c